common/src/main/java/com/google/auto/common/MoreElements.java

```diff
@@ -189,5 +195,24 @@ public static boolean isAnnotationPresent(Element element,
     return Optional.absent();
   }
 
+  /**
+   * Returns a {@link Predicate} that can be used to filter elements by {@link Modifier}.
+   * <p>
+   * Here is an example how one could get a List of static methods of a class:
+   * <code>
+   * FluentIterable.from(ElementFilter.methodsIn(clazzElement.getEnclosedElements()))
+   *     .filter(MoreElements.hasModifiers(Modifier.STATIC).toList();
+   * </code>
+   */
+  public static Predicate<Element> hasModifiers(Modifier... modifiers) {
```

Comment:
I actually would prefer using var args here since it makes for cleaner code:

I think that this is harder to read:
    hasModifers(ImmutableSet.of(Modifier.PUBLIC, Modifier.STATIC));
And would prefer:
    hasModifers(Modifier.PUBLIC, Modifier.STATIC);

Is there any particular harm in terms of using these var args?